RecoLocalTracker/SiPixelClusterizer/plugins/SiPixelRawToClusterGPUKernel.h

```diff
@@ -141,6 +142,7 @@ namespace pixelgpudetails {
     SiPixelRawToClusterGPUKernel& operator=(SiPixelRawToClusterGPUKernel&&) = delete;
 
     void makeClustersAsync(bool isRun2,
+                           bool isHIon,
```

Comment:
My concern with options like `isRun2` and `isHIon` is that it may give the impression we support all combinations of those flags.
Are the two independent, or do we need a specific combination ?

Reply:
in my opinion NONE.
I think I did it once (in the patatrack branch)

Reply:
They are independent. One allows to run on Run2 data (either pp or HI), the other to run on PbPb data.